### ISSUE TITLE
Use django-pipeline to reduce requests for static files

### ISSUE THREAD

**Issue by longhotsummer** [Code4SA/wazimap#3]
The profile page makes:
- 13 requests for javascript files
- 10 requests for css files

The bulk of those requests aren't using Django's staticfiles and so aren't fingerprinted and don't have long cache lifetimes, which means most users will load them repeatedly.

We could probably get those down to 1-2 for each type.

**Comment by xkmato** [comment]
I totally agree with hosting most of them locally - Then we can even leverage some cool tools like django_compressor.

Let me take a shot at this one as it looks like an easy way for me to get introduced into the project.
- I also think introducing bower to manage these would be the reasonable way to go, do you agree?

**Comment by longhotsummer** [comment]
I'm not sure what bower would give us in this case?

Django compressor would be great, perhaps easier to use than django pipeline because it's easier for other projects to add their own css/js.
